### PR TITLE
Book-Store: Styling

### PR DESCRIPTION
**Pull request by degisew**
### Book-Store: Styling
**In this PR, I implemented the following features:**

-  Added a style for the Book page

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -24,22 +24,36 @@ body {
     padding: 10px;
```
Comment (danielufeli) [inline src/index.scss]: - [ ] Beautiful work done in styling your project, kindly align your design to be the same as the one provided see screenshots below.

### Sample


### Your Design

**Review by danielufeli — CHANGES_REQUESTED** [review]

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @degisew,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by degisew** [comment]
Okay, thanks. I will walk through it.

File: src/index.scss [hunk 2]
```diff
@@ -37,11 +35,26 @@ body {
...
+  }
+}
+
+.main-container {
+  background-color: inherit;
+  width: 70%;
+  margin: 20px auto;
+  // box-shadow: 1px 2px 25px rgb(131, 133, 130);
+  min-height: 600px;
+
```
Comment (mwanawabangona) [inline src/index.scss:57]: - [ ] Just as the previous reviewer mentioned make it look like the origin template, 
- [ ] which already your are doing 👏 👏 
- [ ] however please increase the  width `title` `author` and `category` then move it to right like the template 
- [ ] also kindly bookstore heading starts where the first book is starting from.
- [ ] Please see screenshot and highlighted areas for easy identification 
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi, @degisew 

Great work on making the changes requested by a previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.scss (hunk 2 above)
Comment (degisew) [reply src/index.scss:57]: Okay, thanks. I will do it know.

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @degisew      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x] Your project is professional 

- [x]  Your repo is professional 




 

- [x] No Linter Errors





Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._](https://github.com/degisew/Book-Store.git)

**Pull request merged**